cointop/keybindings.go

```diff
@@ -2,173 +2,101 @@ package cointop
 
 import (
 	"strings"
+	"unicode"
 
 	"github.com/cointop-sh/cointop/pkg/gocui"
+	"github.com/gdamore/tcell/v2"
 	log "github.com/sirupsen/logrus"
 )
 
+// keyMap translates key alternative names to a canonical version
+func keyMap(k string) string {
+	key := k
+	switch strings.ToLower(k) {
+	case "lsqrbracket", "leftsqrbracket", "leftsquarebracket":
+		key = "["
+	case "rsqrbracket", "rightsqrbracket", "rightsquarebracket":
+		key = "]"
+	case "space", "spacebar":
+		key = " " // with meta should be "space"
+	case "\\\\", "backslash":
+		key = "\\"
+	case "underscore":
+		key = "_"
+	case "arrowup", "uparrow":
+		key = "Up"
+	case "arrowdown", "downarrow":
+		key = "Down"
+	case "arrowleft", "leftarrow":
+		key = "Left"
+	case "arrowright", "rightarrow":
+		key = "Right"
+	case "return":
+		key = "Enter"
+	case "escape":
+		key = "Esc"
+	case "pageup":
```

Comment:
this is this missing the `"pgup"` alias name

Reply:
This is only for keys that weren't handled natively by tcell (call it "backward compatability").  Tcell has
```
	KeyPgDn:           "PgDn",
	KeyPgUp:           "PgUp",
```

Reply:
ah got it